README.md

```diff
@@ -2,29 +2,122 @@
 
 [![Crates.io](https://img.shields.io/crates/v/iceoryx-rs.svg)](https://crates.io/crates/iceoryx-rs)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Build & Test](https://github.com/eclipse-iceoryx/iceoryx-rs/workflows/Build%20&%20Test/badge.svg?branch=master)](https://github.com/eclipse-iceoryx/iceoryx-rs/actions)
+[![Build & Test](https://img.shields.io/github/workflow/status/eclipse-iceoryx/iceoryx-rs/Build%20&%20Test/master?label=Build%20%26%20Test)](https://github.com/eclipse-iceoryx/iceoryx-rs/actions)
 [![Codecov](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs/branch/master/graph/badge.svg?branch=master)](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs?branch=master)
 
 <p align="center">
 <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
 </p>
 
-Experimental Rust bindings for the [iceoryx](https://github.com/eclipse-iceoryx/iceoryx) IPC middleware.
+Safe Rust bindings to [Eclipse iceoryx](https://github.com/eclipse-iceoryx/iceoryx).
 
-# clone and build
+1. [About](#about)
+2. [Examples](#examples)
+    - [How to start RouDi](#how-to-start-roudi)
+    - [Run the simple publisher and subscriber example](#run-the-simple-publisher-and-subscriber-example)
+3. [Limitations](#limitations)
 
-Clone the repository with
+## About
+
+Eclipse iceoryx is a true zero-copy, inter-process communication framework with the goal to boost
+autonomous driving with their demand on high data throughput and low latency. With this properties,
```

Comment:
I heard the suggestion lately to not state `this` or `mentioned earlier` or phrases like that. One should really name the things again.

With this in mind you could also write `With iceoryx bandwidth and speed, is also fits ...`